            #prob..01

'''name=input("enter a name:")
print("good ofternoon"+name)'''

             #prob no..02

'''
letter="  Dear <!NAME!>,
                 you are selected!
                   <!DATE!>"
name=input("enter a name\n:")
date=input("enter a date\n:")
letter=(letter.replace('<!NAME!>',name))
letter=(letter.replace('<!DATE!>',date))
print(letter)'''
                #prob..no..03

story = '''In the world,a country is wonderful thats is bharat.
in the bharat a state thats called the heart  of bharat  is madhya pradesh '''
doublspacec=story.find("  ")
print(doublspacec)

                #prob.no..04

story = '''In the world,a country is wonderful thats is bharat.
in the bharat a state thats called the heart  of bharat  is madhya pradesh '''
story=(story.replace("  "," "))
print(story)

                #prob.no..04

story = '''In the world,a country is wonderful thats is bharat.
in the bharat a state thats called the heart  of bharat  is madhya pradesh '''
formated_letter='''In the world\n\t,a country is wonderful thats is bharat\n.
in the bharat a state thats called the "heart  of bharat"  is "madhya pradesh" '''
print(formated_letter)
